lib/rubycritic/commands/default.rb

```diff
@@ -12,7 +13,15 @@ def initialize(paths)
       end
 
       def execute
-        report(critique)
+        case Config.format
+        when :json
+          report(critique)
+        when :html
+          report(critique)
+          open
+        else
+          raise "Unknown output format #{Config.format}"
```

Comment:
This will now crash on our newly merged "ConsoleReport", won't it?
Did you rebase against master?